docs/sources/administration/service-accounts/about-service-accounts.md

```diff
@@ -0,0 +1,46 @@
+---
+title: About service accounts
+menuTitle: About service accounts
+aliases: [docs/sources/manage-service-accounts/_index.md, 
+docs/sources/administration/service-accounts/about-service-accounts.md,
+docs/sources/administration/service-accounts/create-service-accounts.md]
+description: "This contains detailed information about service accounts in Grafana"
+weight: 30
+---
+
+# About service accounts in Grafana
+
+A service account is a regular user account used to run automated or compute workloads which are intended for programmatic use either internally for a Grafana feature or to communicate outside of the Grafana environment. Applications use service account tokens to authorize themselves as a service account.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up to manage users and permissions across the organization to an external auth provider
+- Machine to machine communication
+
+It can also be used in combination with FGAC to grant specific scopes.
+
```

Comment:
This first sentence is passive voice and we want to write in the active voice. In this case, active voice is "You can associate a service account with multiple API keys." When would a user associate a service account with multiple API keys? Because you've introduced that this can be done, you should provide some examples/use cases of when to do it, or at least some benefits.

Reply:
gotcha. more direct and introduce examples perhaps